sympy/functions/elementary/piecewise.py

```diff
@@ -276,6 +282,25 @@ def eval(cls, *_args):
 
             current_cond.add(cond)
 
+            #   Does not add intervals which are already covered entirely by
+            #   previous intervals (in terms of ordering of arguments)
+            #   Piecewise((0, t>=4), (0, t<3), (1, t<=2), (3, t<4))
+            #   >>> Piecewise((0, (t >= 4) | (t < 3)), (3, True))
+            #   (Feature implemented for 1 variable Piecewise only)
+            if(len(symbols)<2):
```

Comment:
The items of `symbols` are sets and `symbols[0]` may contain many variables.

Reply:
@jksuom I modified it to include your concern, but I think the `try:` block was taking care of it anyways.